tests/media.md

```diff
@@ -0,0 +1,36 @@
+# Notebooks for all media
+
+A collection of multimedia references to Notebooks for All.
+
+## Articles 
+
+* [Making Jupyter Notebooks More Accessible]
+* referenced in [Data Driven Understanding of Data
+Science Notebook Inaccessibility]
+* [SciPy 2023](https://gihyo.jp/article/2023/08/scipy2023) written in Japanese
+
+## Videos
+
+* [Patrick Smyth, Jenn Kotler Notebooks for All: Accessibility & Jupyter Notebooks | JupyterCon 2023](https://www.youtube.com/watch?v=LJMS9R-KFXQ)
+* [Basic Jupyter Notebook Accessibility](https://www.youtube.com/watch?v=KsUF_HjA97U&t=318s)
+* [arXiv Accessibility Forum 2023 — Dr. Patrick Smyth: Hackccessibility for All](https://www.youtube.com/watch?v=1v3jFCWQfI8)
+* [STScI Day of Accessibility playlist](https://www.youtube.com/playlist?list=PLCPZgcYzVpj_WHHCTUpec8THYEMzXZnR1)   
+    * [Introduction to Day of Accessibility]
+    * [Notebooks for All]
+    * [Illuminating the Universe with Accessible Text Descriptions]
+    * [Joshua Miele - Day of Accessibility Keynote]
+    * [Hearing the Light - Astronomy Data Sonification]
+    * [Learn to Use a Screen Reader]
+* [Accessibility best practices for authoring Jupyter notebooks] @ JupyterCon
```

Comment:
Since I see another JupyterCon talk on this list and this is the name of the SciPy talk, I think this is what is meant.

```suggestion
* [Accessibility best practices for authoring Jupyter notebooks - SciPy 2023](https://youtu.be/onjRLjveGxk)
```

Reply:
Correcting to match Markdown reference formatting but keep more accurate title.

```suggestion
* [Accessibility best practices for authoring Jupyter notebooks - SciPy 2023]
```